Makefile

```diff
@@ -280,9 +280,12 @@ deploy-ocm-secret: deploy-namespace
 	python3 ./tools/deploy_sso_secret.py --secret "$(OCM_CLIENT_SECRET)" --id "$(OCM_CLIENT_ID)" --namespace "$(NAMESPACE)" \
 		--target "$(TARGET)" --apply-manifest $(APPLY_MANIFEST)
 
-deploy-image-service: deploy-namespace
+deploy-image-service: deploy-image-service-file
 	python3 ./tools/deploy_image_service.py --namespace "$(NAMESPACE)" --apply-manifest $(APPLY_MANIFEST)
 
+deploy-image-service-file: deploy-namespace
+	python3 ./tools/deploy_image_service_service.py --target "$(TARGET)" --namespace "$(NAMESPACE)" --apply-manifest $(APPLY_MANIFEST)
+
```

Comment:
I don't see why we need separate targets for this. Couldn't one python script deploy the service, wait for an IP, then deploy the deployment?

I don't see why we would ever want to use one and not the other.

Reply:
yeah just followed assisted-service for consistency, merged into a single one (deploy_image_service.py).